input/fsh/EX_gdx-example02.fsh

```diff
@@ -5,12 +5,12 @@
 // **************************************************************
 // ************* Example 2: Tumor-Normal Test *******************
 // **************************************************************
-Instance: gx2-patient-srikar-chamala
+Instance: gx2-patient-john-doe
 InstanceOf: GdxPatient
 Description: "GenomeX Example2 of NGS Patient"
 * identifier.use = #usual
 * name.family = "Chamala"
```

Comment:
Did you want to use 'Doe' as the family name?

Reply:
good catch. lemme fix.